src/lib/util/util.js

```diff
@@ -98,6 +99,182 @@ class Util {
 		}
 	}
 
+	/**
+	 * Get the type of value. A better version of the `typeof` operator, basically
+	 * @param {*} value The object or primitive whose type is to be returned
```

Comment:
Can you do me the favor of including a `@since 0.4.0` tag for all methods you are adding.